src/includes/getting-started-config/javascript.capacitor.mdx

```diff
@@ -0,0 +1,53 @@
+You should `init` the SDK as soon as possible during your app's creation stage and forward the `init` method from the sibling Sentry SDK for the framework you use.
+
+```typescript {tabTitle: Angular} {filename: app.module.ts}
+import * as Sentry from "@sentry/capacitor";
+import * as SentryAngular from "@sentry/angular";
+
+Sentry.init(
+  {
+    dsn: "___PUBLIC_DSN___",
+
+    // To set your release and dist versions
+    release: "my-project-name@" + process.env.npm_package_version,
```

Comment:
@bruno-garcia This was a carry-over from the previous version of this page. I think it would make more sense to just put it as `<release>` given we refer to it that way in the source map upload command below?

Reply:
or maybe `"my-project-name@" + <release_name>`

Reply:
Sounds good!